README.md

```diff
@@ -16,7 +16,13 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+If an image is too large to fit into the element containing it.
+
+Example:
+
+.clearfix{
+overflow: auto;
+}
 ```
```

Comment:
Clearfix is a CSS class selector used to fit floated elements inside of a container. The size of the element (whether it's an image or otherwise) doesn't come into play here. 

Check this blog post out if you need a refresher on floats and clearing: https://css-tricks.com/all-about-floats/